File: Go/String/2675.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	var input string
	var roop int
	var tc int
	reader := bufio.NewReader(os.Stdin)
	fmt.Scanf("%d", &tc)

	for i := 0; i < tc; i++ {
		fmt.Fscanln(reader, &roop, &input)
		for _, val := range input {
			for i := 0; i < roop; i++ {
				fmt.Print(string(val)) //(rune to string)
			}
		}
		fmt.Println()
	}

}

File: python/String/2675.py
import sys


if __name__ == "__main__":
    tc = int(input())

    for i in range(tc):
        in_list = list(map(str, input().split(" ")))

        roop = int(in_list[0])
        char = in_list[1]

        for val in char:
            print(val*roop, end='')

        print()   
